packages/go/cypher/models/pgsql/test/translation_cases/pattern_expansion.sql

```diff
@@ -337,3 +337,92 @@ with s0 as (with recursive ex0(root_id, next_id, depth, satisfied, is_cycle, pat
+                                                                                              e0.start_id = e0.end_id,
+                                                                                              array [e0.id]
+                                                                                       from edge e0
+                                                                                              join node n0 on
+                                                                                         n0.kind_ids operator (pg_catalog.&&)
+                                                                                         array [1]::int2[] and
+                                                                                         n0.properties ->>
+                                                                                         'objectid' like '%1234' and
+                                                                                         n0.id = e0.start_id
+                                                                                              join node n1 on n1.id = e0.end_id
+                                                                                       where e0.kind_id = any (array [11, 12]::int2[])
+                                                                                       union
+                                                                                       select ex0.root_id,
+                                                                                              e0.end_id,
+                                                                                              ex0.depth + 1,
+                                                                                              n1.kind_ids operator (pg_catalog.&&)
+                                                                                              array [1]::int2[] and
+                                                                                              n1.properties ->>
+                                                                                              'objectid' like '%4567',
+                                                                                              e0.id = any (ex0.path),
+                                                                                              ex0.path || e0.id
+                                                                                       from ex0
+                                                                                              join edge e0 on e0.start_id = ex0.next_id
+                                                                                              join node n1 on n1.id = e0.end_id
+                                                                                       where ex0.depth < 5
```

Comment:
Same here re: unbounded variable paths

Reply:
Noted. I'd like to come back to this in another changeset.